<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>627</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Изучаем системы счисления. V 1.2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="study">
       <attribute name="title">
        <string>Обучение</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="problem">
          <property name="text">
           <string>Введите значение числа</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="QLabel" name="remark">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="solution"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="task">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>401</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="verify">
          <property name="text">
           <string>Проверить</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="skip">
          <property name="text">
           <string>Следующее число</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="4">
         <widget class="QLabel" name="ver">
          <property name="text">
           <string>Статистика выполнения</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statistic">
       <attribute name="title">
        <string>Статистика</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableUser">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Пользователь</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Уровень</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Попыток</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Успешно</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>01.01.2021</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>7</string>
           </property>
           <property name="textAlignment">
            <set>AlignTrailing|AlignVCenter</set>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>77</string>
           </property>
           <property name="textAlignment">
            <set>AlignTrailing|AlignVCenter</set>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string>3</string>
           </property>
           <property name="textAlignment">
            <set>AlignTrailing|AlignVCenter</set>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="preferences">
       <attribute name="title">
        <string>Настройка</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox_droby">
          <property name="text">
           <string>Использовать дробные числа</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_all_syst">
          <property name="text">
           <string>2-, 4-, 8-, 16-ричные системы счисления</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton_2_4_8">
          <property name="text">
           <string>Двоичная, четверичная, восьмеричная системы счисления</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_bin_only">
          <property name="text">
           <string>Только двоичная система счисления</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="teacher">
       <attribute name="title">
        <string>Учителю</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_export_stat">
          <property name="text">
           <string>Выгрузить статистику</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_new_user">
          <property name="text">
           <string>Добавить ученика</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_clear_stat">
          <property name="text">
           <string>Очистить статистику</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tableAllUser">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Учащийся</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Уровень</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Попыток</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Успешно</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>01.01.2021</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>7</string>
           </property>
           <property name="textAlignment">
            <set>AlignTrailing|AlignVCenter</set>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>77</string>
           </property>
           <property name="textAlignment">
            <set>AlignTrailing|AlignVCenter</set>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string>3</string>
           </property>
           <property name="textAlignment">
            <set>AlignTrailing|AlignVCenter</set>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>За сегодня</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>За все время</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>solution</tabstop>
  <tabstop>verify</tabstop>
  <tabstop>skip</tabstop>
  <tabstop>tableUser</tabstop>
  <tabstop>radioButton_bin_only</tabstop>
  <tabstop>radioButton_2_4_8</tabstop>
  <tabstop>radioButton_all_syst</tabstop>
  <tabstop>checkBox_droby</tabstop>
  <tabstop>tableAllUser</tabstop>
  <tabstop>pushButton_new_user</tabstop>
  <tabstop>pushButton_export_stat</tabstop>
  <tabstop>pushButton_clear_stat</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
